private_gpt/__main__.py

```diff
@@ -5,7 +5,8 @@
 from private_gpt.main import app
 from private_gpt.settings.settings import settings
 
-# Set log_config=None to do not use the uvicorn logging configuration, and
-# use ours instead. For reference, see below:
-# https://github.com/tiangolo/fastapi/discussions/7457#discussioncomment-5141108
-uvicorn.run(app, host="0.0.0.0", port=settings.server.port, log_config=None)
+if __name__ == "__main__":
```

Comment:
Only by convenience on my end (to click on the little ⏯️ button in my PyCharm) - sorry, will remove

On another note, for `__main__.py`, this block will always be evaluated, given that `__name__` will always return `__main__`, per the filename.

Otherwise, `if __name__ == "__main__":` is the common pattern to run code conditionally on how the python file get invoked (just like in `private_gpt/ui/ui.py`)